core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.html

```diff
@@ -51,8 +51,10 @@
           <span ng-if="!topic.is_published"> Not Published </span>
         </a>
       </td>
-      <td ng-if="canDeleteTopic()" class="delete-icon">
-        <span ng-click="deleteTopic(topic.id)" class="fas fa-trash-alt list-summary protractor-test-delete-topic-button"> </span>
+      <td class=" delete-icon">
```

Comment:
remove the space from the beginning. Also, have you check the UI when the user can't delete the topic i.e, to check whether moving `ng-if` from <td>` tag to anchor tag doesn't break the UI.)

Reply:
Done. I checked by modifying the ng-if="!canDeleteTopic()" and this was the result:
![Screenshot from 2020-04-14 20-55-41](https://user-images.githubusercontent.com/42586894/79255303-af0f3580-7ea3-11ea-9d58-f5d915b07c5c.png)

Reply:
Awesome, the (update) image LGTM! :)